### PR TITLE
INSIGHTS-207 - [Plugin] Add OPA imports (Vertex)

### PR DESCRIPTION
**Pull request by vitorvezani**
This PR fixes internal INSIGHTS-207

## Checklist
* [X] I have signed the CLA
* [X] I have updated/added any relevant documentation

### What's the goal of this PR?
- [OPA] add support for OPA custom libs
- [admission] add support for OPA custom libs
- [CI] add support for OPA custom libs
- [admission] remove packr2 from build (no longer required for Polaris)

### What alternative solution should we consider, if any?

[Internal Ticket INSIGHTS-207](https://fairwinds.myjetbrains.com/youtrack/issue/INSIGHTS-207)

### CODE REVIEW

File: plugins/opa/pkg/opa/opa.go [hunk 1]
```diff
@@ -54,11 +49,12 @@ func Run(ctx context.Context) ([]ActionItem, error) {
 
 // processAllChecks runs the supplied slices of OPACustomCheck and
 // CheckSetting and returns a slice of action items.
-func processAllChecks(ctx context.Context, checkInstances []CheckSetting, checks []OPACustomCheck, thisNamespace string) ([]ActionItem, error) {
```
Comment (vitorvezani) [inline plugins/opa/pkg/opa/opa.go:57]: `thisNamespace` was not used

File: plugins/admission/go.mod [hunk 2]
```diff
@@ -7,7 +7,7 @@ replace github.com/fairwindsops/insights-plugins/plugins/admission => ./
 replace github.com/fairwindsops/insights-plugins/plugins/opa => ../opa
 
 require (
-	github.com/fairwindsops/insights-plugins/plugins/opa v0.0.0-20240214152350-40f728c8f865
+	github.com/fairwindsops/insights-plugins/plugins/opa v0.0.0-00010101000000-000000000000
```
Comment (vitorvezani) [inline plugins/admission/go.mod:10]: This don't really matter, as it uses the `replace` above

File: plugins/opa/pkg/opa/opa.go [hunk 3]
```diff
@@ -32,20 +33,11 @@ var instanceGvr = schema.GroupVersionResource{Group: "insights.fairwinds.com", V
 var checkGvr = schema.GroupVersionResource{Group: "insights.fairwinds.com", Version: "v1beta1", Resource: "customchecks"}
 
 func Run(ctx context.Context) ([]ActionItem, error) {
-	thisNamespace := "insights-agent"
```
Comment (vitorvezani) [inline plugins/opa/pkg/opa/opa.go:35]: `thisNamespace` was not used

File: plugins/opa/results.schema [hunk 4]
```diff
@@ -31,10 +31,7 @@
             },
             "Category": {
               "type": "string"
-            },
```
Comment (vitorvezani) [inline plugins/opa/results.schema:34]: duplicated from line 14

File: plugins/admission/install-and-run-packr2-for-polaris.sh
Comment (vitorvezani) [inline plugins/admission/install-and-run-packr2-for-polaris.sh:1]: packer2 is not needed anymore for polaris after rework

**Review by jdesouza — APPROVED** [review]

**Pull request merged**